modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -481,7 +491,32 @@ using GConstGIEModel = ade::ConstTypedGraph
     , IECallable
     >;
 
+inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
```

Comment:
I really wonder how it works with ROI (assuming `infer-roi` or `infer-list`)

Reply:
For now I would merge it like this. We can test it separately and fix if needed. Created a task on that